vcr/pe/store.go

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2023 Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package pe
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+)
+
+// Store is a store for presentation definitions
+// It loads a file with the mapping from oauth scope to presentation definition
+type Store struct {
```

Comment:
This is now a "presentation exchange store" (since it's in the `pe` package), which is a confusing name. Maybe "DefinitionStore" is better? Or a `DefinitionResolver` with  a

```
type DefinitionResolver interface {
     ByScope(string) *PresentationDefinition
}

type DefinitionMap map[string]PresentationDefinition
```

type which had the `ByScope` function? Then the LoadFromFile and LoadFromBytes can simply we top-level functions, no need to make them methods on the type, I think?

Reply:
There's no need for an interface